sharding/shardset.go

```diff
@@ -148,7 +148,19 @@ func validateShards(shards []shard.Shard) error {
 
 // DefaultHashGen generates a HashFn based on murmur32
 func DefaultHashGen(length int) HashFn {
```

Comment:
Maybe we should rename this to `DefaultHashFn(length int) HashFn` as it returns a `HashFn` not a `HashGen`.

Reply:
sure thing